app/views/interactions/show.html.haml

```diff
@@ -0,0 +1,56 @@
+=content_for :title, "DGIdb - #@title Interaction Record"
+=content_for :header do
+	%h2
```

Comment:
The indentation in this file is off (too far indented). The standard is two spaces per indentation level.